axes/utils.py

```diff
@@ -1,17 +1,32 @@
+import logging
+
 from platform import python_version
 from sys import platform
+
```

Comment:
Please include style changes in a separate PR.

Reply:
Sure no problem.